fabric-registry-sync-v0/src/main/resources/assets/fabric-registry-sync-v0/lang/en_us.json

```diff
@@ -0,0 +1,7 @@
+{
+  "fabric-registry-sync-v0.unknown-remote.title.singular" : "Received a registry entry that is unknown to this client.\n",
+  "fabric-registry-sync-v0.unknown-remote.title.plural" : "Received %d registry entries that are unknown to this client.\n",
+  "fabric-registry-sync-v0.unknown-remote.subtitle.1" : "This is usually caused by a mismatched mod set between the client and server.",
```

Comment:
Should we explicitly mention modpack version mismatches as a possible candidate? Maybe
`, for example caused by different modpack versions.`

Reply:
Modpacks could modify this message themselves, allowing for more precise errors.

Reply:
I think we are slightly pushed for space here (Likely even more so once translated) it cannot really go onto another line. @Patbox 's idea is good, nothing stopping a modpack from changing this 👍

Reply:
I think `mismatched mod set` is a bit too technical/abstract for some users, hence my suggestion. Worst case we show one less namespace.